src/usecases/deleteVisitByCallId.js

```diff
@@ -0,0 +1,28 @@
+const deleteVisitByCallId = ({ getDb }) => async (callId) => {
+  const db = await getDb();
+  console.log("deleting visit for  ", callId);
+  try {
+    const results = await db.any(
+      `DELETE FROM scheduled_calls_table WHERE call_id = $1 LIMIT 1`,
+      callId
+    );
+
+    const result = results[0];
+    console.log("result", result);
+
+    return {
+      result: {
+        success: true,
+      },
+      error: null,
+    };
+  } catch (error) {
+    console.error(error);
+    return {
+      result: null,
+      error: error.toString(),
+    };
```

Comment:
If the above suggestion is the way we want to go, then we can updated this to `{ success: false, error: error.toString() }`.

Reply:
Agree :)